sdk-springboot/src/main/java/io/dapr/springboot/DaprController.java

```diff
@@ -27,28 +29,65 @@
    */
   private static final DefaultObjectSerializer SERIALIZER = new DefaultObjectSerializer();
 
+  /**
+   * Callback API for health checks from Dapr's sidecar.
+   */
+  @GetMapping("/healthz")
+  public void healthz() {

```

Comment:
can you help me understand how and where does this results in a  200 OK response?

Reply:
It will handle a get request on /healthz, do nothing and return nothing. So, it is a successful request for Springboot = 200.